enhancements/builds/entitlements.md

```diff
@@ -0,0 +1,295 @@
+---
+title: openshift-builds-improve-use-of-rhel-entitlements
+
+authors:
+  - "@gabemontero"
+  - "@bparees" (provided a lot of input in tracking Jira)
+
+reviewers:
+  - "@bparees"
+  - "@adambkaplan"
+  - "@who else?"
```

Comment:
@zvonkok should track this as well as he will be a consumer.

Reply:
I've added him thanks.